basic_bootloader/src/bootloader/transaction/rlp_encoded/mod.rs

```diff
@@ -170,4 +191,12 @@ mod test {
             tx.hash_for_signature_verification().as_u8_array()
         )
     }
+
+    #[test]
+    fn test_on_random_service() {
```

Comment:
Should we also check a negative case? Can be useful as a regression test

Reply:
We have negative tests in the service_tx module, what regression are you expecting to catch?